src/mmw/apps/modeling/views.py

```diff
@@ -102,18 +105,32 @@ def _initiate_tr55_job_chain(model_input, job_id):
 @decorators.api_view(['GET'])
 @decorators.permission_classes((AllowAny, ))
 def district(request, id=None, state=None):
-    if id:  # query by unique id
-        district = get_object_or_404(District, id=id)
-        coordinates = []
-        for polygon in district.polygon.coords:
-            coordinates.append(polygon[0])
-        dictionary = {'type': 'Feature',
-                      'properties': {},
-                      'geometry': {'type': 'Polygon',
-                                   'coordinates': coordinates}}
-        return Response(dictionary)
-    else:  # provide list of all ids
-        shapes = District.objects.order_by('state_fips', 'district_fips')
-        shapes = [{'id': shape.id, 'name': shape.name()} for shape in shapes]
-        dictionary = {'shapes': shapes}
-        return Response(dictionary)
+    district = get_object_or_404(District, id=id)
+    coordinates = []
+    for polygon in district.geom.coords:
```

Comment:
Consider separating this out into a utility function to explain what's going on here.